### PR TITLE
coverity 1230692 : fix constructor

### PR DESCRIPTION
**Pull request by dragon512**

### CODE REVIEW

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/521/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/393/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/398/

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/409/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/2092/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/1985/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/654/

**Comment by dragon512** [comment]
Don't merge... have another update to this coming

**Comment by dragon512** [comment]
ok .. should be good now

**Comment by atsci** [comment]
RAT check *failed*! https://ci.trafficserver.apache.org/job/RAT-github/425/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/414/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/416/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/538/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2002/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/2108/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/411/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2004/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/669/

File: lib/ts/HashSip.cc [hunk 1]
```diff
@@ -33,24 +35,18 @@ Based off of implementation:
   x3 ^= x0;                         \
   x2 = ROTL64(x2, 32);
 
-ATSHash64Sip24::ATSHash64Sip24()
+ATSHash64Sip24::ATSHash64Sip24() : k0(0), k1(0)
```
Comment (zwoop) [inline lib/ts/HashSip.cc]: i think this is superfluous.

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/559/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/433/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/690/

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/444/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2033/

**Comment by atsci** [comment]
FreeBSD11 build *failed*! https://ci.trafficserver.apache.org/job/freebsd-github/2137/

**Pull request merged**